docs/node/mainnet/eden-upgrade.md

```diff
@@ -214,15 +214,15 @@ Ubuntu versions (or other distributions using older system libraries).
 [systemd]: https://systemd.io/
 [Supervisor]: http://supervisord.org/
 [Wipe state]: ../run-your-node/maintenance/wiping-node-state.md#state-wipe-and-keep-node-identity
-[23.0.3]: https://github.com/oasisprotocol/oasis-core/releases/tag/v23.0.3
+[23.0.x]: https://github.com/oasisprotocol/oasis-core/releases/tag/v23.0.x
 [rosetta-gw-2.6.0]:
   https://github.com/oasisprotocol/oasis-rosetta-gateway/releases/tag/v2.6.0
 
 ### Configuration Changes
 
 :::info
 
-To see the full extent of the changes examine the [Change Log] of the 23.0.3
+To see the full extent of the changes examine the [Change Log] of the 23.0.x
```

Comment:
Here, we probably want audience to read about changes that the new major version of Oasis Core brought and not the specific patch version, right?

```suggestion
To see the full extent of the changes examine the [Change Log] of the 23.0
```